inc/sainexthop.h

```diff
@@ -149,6 +149,26 @@ typedef enum _sai_next_hop_attr_t
      */
     SAI_NEXT_HOP_ATTR_TUNNEL_ID,
 
+    /**
+     * @brief Next hop entry VNI
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 0
+     * @validonly SAI_NEXT_HOP_ATTR_TYPE == SAI_NEXT_HOP_TYPE_TUNNEL_ENCAP
+     */
+    SAI_NEXT_HOP_ATTR_TUNNEL_VNI,
+
+    /**
+     * @brief Inner destination MAC address
+     *
+     * @type sai_mac_t
+     * @flags CREATE_AND_SET
+     * @default attrvalue SAI_SWITCH_ATTR_VXLAN_DEFAULT_ROUTER_MAC
```

Comment:
SAI_SWITCH_ATTR_VXLAN_DEFAULT_ROUTER_MAC is not defined anywhere

Reply:
combined with #751